src/reducers/cityReducer.js

```diff
@@ -1,13 +1,18 @@
 import * as types from './../actions/types';
 
 export default function cityReducer(state = { name: '' }, action) {
-  switch (action.type) {
+    switch (action.type) {
     case types.SELECT_CITY:
-      return {
-        ...state,
-        name: action.city,
-      };
+        return {
+            ...state,
+            name: action.city,
+        };
+    case types.SAVE_LOCALISATION:
```

Comment:
Do you think that we should be keep it together ? what about moving it to separate reducer?